docs/quality-assurance-test-cases.md

```diff
@@ -173,6 +173,21 @@ This document contains some examples of test cases for each feature implemented
 | Logged in user tries to complete a task from an non existing request (The request  exists in a different relationship) | Fail |
 | Logged in user tries to complete a task from a request which is not in the ACCEPTED state (as a mentor or as a mentee) | Fail |
 
+### List All Tasks
+**Service:** PUT /mentorship_relation/{request_id}/task/{task_id}/complete
+
+| Test Case      | Outcome |
```

Comment:
@khushishikhu  These test cases according to me, aren't valid for list all tasks.
This is not similar to what is given as a mock in PR.
@khushishikhu The state of task shouldn't be included in any condition.
Only the owner (mentor/mentee) of a task should be the primary condition to check if the task belongs to the user, if yes, it is returned in the list.
For eg.

Logged in user requests list all tasks, and out of all tasks, those which involve user (as a mentor or as a mentee) - **Success**
and the other one being where user isn't involved - **Fail**

Reply:
you mean for listing tasks , there should not be a condition of ACCEPTED task or REJECTED task